src/ln/peer_handler.rs

```diff
@@ -710,12 +721,32 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 						}
 						continue;
 					},
+					Event::HandleOutboundError { ref node_id, ref err } => {
+						let (mut descriptor, peer) = get_peer_for_forwarding!(node_id, {
+								//TODO: Do whatever we're gonna do for handling dropped messages
```

Comment:
Note duplicate indentation here, and also probably don't need a TODO, just note that we don't have to disconnect them if they're already disconnected.